backend/src/middleware/notifications/notificationsMiddleware.spec.js

```diff
@@ -88,51 +97,72 @@ describe('notifications', () => {
             id
             content
           }
+          ... on Report {
+            id
+            filed {
+              reasonCategory
+              reasonDescription
+              reportedResource {
+                __typename
+                ... on User {
+                  id
+                  name
+                }
+                ... on Post {
+                  id
+                  content
+                }
+                ... on Comment {
+                  id
+                  content
+                }
+              }
+            }
+          }
         }
       }
     }
   `
 
+  let title
+  let postContent
+  let postAuthor
+  const createPostAction = async () => {
+    authenticatedUser = await postAuthor.toJson()
+    await mutate({
+      mutation: createPostMutation,
+      variables: {
+        id: 'p47',
+        title,
+        postContent,
+        categoryIds,
+      },
+    })
+    authenticatedUser = await notifiedUser.toJson()
+  }
+
+  let commentContent
+  let commentAuthor
+  const createCommentOnPostAction = async () => {
+    await createPostAction()
+    authenticatedUser = await commentAuthor.toJson()
+    await mutate({
+      mutation: createCommentMutation,
+      variables: {
+        id: 'c47',
+        postId: 'p47',
+        commentContent,
+      },
+    })
+    authenticatedUser = await notifiedUser.toJson()
+  }
+
   describe('authenticated', () => {
     beforeEach(async () => {
       authenticatedUser = await notifiedUser.toJson()
     })
 
     describe('given another user', () => {
-      let title
-      let postContent
-      let postAuthor
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2020-01-23T18:35:19Z" title="Thursday, January 23rd 2020, 7:35:19 pm +01:00">Jan 23, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -541,5 +573,148 @@ describe('notifications', () => {
         })
       })
     })
+
+    describe('given me filing a report of a', () => {
```


```suggestion
    describe('given I file a report on a', () => {
```